javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java

```diff
@@ -139,11 +122,11 @@ private void insertCommentsInNode(Node node,
             }
         }
 
-        // at this point I create an ordered list of all remaining comments and
-        // children
+        /* at this point I create an ordered list of all remaining comments and
```

Comment:
Line comment? Capital letter for the comment?

(I must have found my nitpick-hat)